### PR TITLE
Make dhcp-server build

### PR DESCRIPTION
**Pull request by vmoravec**
- resolve excluded files
- include patch file

### CODE REVIEW

File: patches/dhcp-server.patch [hunk 1]
```diff
@@ -0,0 +1,741 @@
...
+ y2milestone ("----------------------------------------");
+ y2milestone ("Dhcp-server module started");
+ 
+-import "DhcpServerUI";
+-
+-/**
+-  * CommandLine handler for running GUI
+-  * @return boolean true if settings were saved
+-  */
+-define boolean GuiHandler () {
```
Comment (jreidinger) [inline patches/dhcp-server.patch]: this looks definitively wrong for to to remove function from code

File: patches/dhcp-server.patch [hunk 2]
```diff
@@ -0,0 +1,741 @@
...
+ textdomain "dhcp-server";
+ 
+ include "dhcp-server/misc.ycp";
+ 
++
++/**
++  * CommandLine handler for running GUI
++  * @return boolean true if settings were saved
++  */
++define boolean GuiHandler () {
```
Comment (jreidinger) [inline patches/dhcp-server.patch]: I don't like such movement, it changes structure of ycp code and also as sequence structure of rby code. I prefer to keep it excluded for this case.

Comment (vmoravec) [inline patches/dhcp-server.patch]: I moved this code after discussion with @lslezak , is excluding the only correct way of handling that case?

Comment (jreidinger) [inline patches/dhcp-server.patch]: It depends on our strategy for compiling included modules, but for beginning I prefer to not modify too much and add such patches only if needed.

**Comment by vmoravec** [comment]
Updated as discussed above

**Pull request merged**